README.md

```diff
@@ -493,6 +519,10 @@ highlight! CmpItemKindMethod guibg=NONE guifg=LightYellow
 
 The menu field.
 
+#### `CmpItemMenuThumb`
+
+The scrollbar highlighting for the `scrollbar`, when `scrollbar` is not `""`.
```

Comment:
I think we already have `PmenuThumb` and `PmenuSbar`.

Reply:
We need a different highlight group when the scrollbar is actually a character. When it is an empty string, the background color typically determines how the scrollbar looks as there are no characters in the buffer. When it is a specific character, the foreground is used.

If we use `PmenuThumb`, you'll get behavior like [this](https://github.com/hrsh7th/nvim-cmp/pull/472#issuecomment-967777691). If we use this highlight group, you get [this](https://github.com/hrsh7th/nvim-cmp/pull/472#issuecomment-968200905).